frontend_tests/node_tests/message_list_data.js

```diff
@@ -195,6 +198,7 @@ run_test('more muting', () => {
         bottom_messages: [
             {id: 10, topic: 'whatever'},
         ],
+        newest_messages: [],
     });
 
 });
```

Comment:
We should probably write some proper tests for the corner cases around this new feature, since it's somewhat tricky.